packages/mobile/src/screens/chat-screen/ChatScreen.tsx

```diff
@@ -416,23 +417,25 @@ export const ChatScreen = () => {
                 placeholder={messages.startNewMessage}
                 Icon={() => (
                   <IconSend
-                    fill={palette.primary}
                     width={styles.icon.width}
                     height={styles.icon.height}
                     opacity={iconOpacity}
+                    fill={styles.icon.fill}
                     onPress={() => handleSubmit(inputMessage)}
                   />
                 )}
                 styles={{
                   root: styles.composeTextContainer,
-                  input: styles.composeTextInput
+                  input: [
+                    styles.composeTextInput,
+                    Platform.OS === 'ios' ? { paddingBottom: spacing(1) } : null
```

Comment:
that's super interesting 👀 

for reference, on Web, we "fake" the input and have it transparent and set the lineheight exactly for one line of text, and then have the containing element have padding. Not suggesting that would work here, but calling out in case it's useful